platform/README.md

```diff
@@ -203,8 +203,8 @@ but doesn't actually return the character.  It's only required if you use the KE
 If your hardware requires you to read the character while checking whether one is
 ready, you should buffer it and make sure that `kernel_getc` checks the buffer.
 (See `platform-py65mon.asm` as one example.)
-If your hardware doesn't support kbhit at all, you can use this dummy
-implementation that makes KEY? behave like the blocking KEY:
+If your hardware doesn't support kbhit, or you don't care about KEY?, taliforth.asm
+will provide a default implementation that makes KEY? block like KEY:
```

Comment:
The default (weak) implementation for `kernel_kbhit` doesn't block at all, but rather it simply always returns true, right?

Reply:
true, I guess I actually mean that the subsequent KEY call will end up blocking.  i'll clarify that